commands/ssh.js

```diff
@@ -15,7 +15,9 @@ module.exports = function(topic, command) {
     topic: topic,
     command: command,
     description: 'Create an SSH session to a dyno',
-    help: `Usage: heroku ${topic}:${command}`,
+    help: `Example:
+    
+     heroku ${topic}:${command}`,
```

Comment:
why is the indentation different on this one? Also, all of these should start with `$`